cmd/project/project_worker.go

```diff
@@ -102,14 +127,20 @@ func init() {
 	projectWorkerCmd.PersistentFlags().String("queue", "", "Queues to consume")
 	projectWorkerCmd.PersistentFlags().String("memory-limit", "", "Memory Limit")
 	projectWorkerCmd.PersistentFlags().String("time-limit", "", "Time Limit")
+	projectWorkerCmd.PersistentFlags().Uint("graceful-stop-limit", 0, "Graceful Stop Limit")
```

Comment:
maybe set a default value where it is enabled like 5?
what do you think?

Reply:
My thinking was that it's better to have this disabled by default, because
1. this (SIGKILL) is how it should have been working since the start and it only didn't because the context propagation was not working
2. pcntl is not enabled by default in PHP (according to docs), so there is no difference between SIGINT/SIGTERM and SIGKILL in many cases

On the other hand, all PHP distributions I know enable pcntl and it would make a lot of sense for me to enable this by default, because when would you want to hard abort a queue message.

I'd set this to 30 seconds by default, because then we can be reasonably sure that a message can actually be handled in that time. In most cases, the messenger stops far quicker, so you will probably never wait for the whole time limit.